learn-beta2.md

```diff
@@ -46,6 +38,10 @@ A series of guides on various things you need to do during the Ballerina develop
 
 Presentation slides on the Ballerina language that anyone can use to talk about the language.
 
+### [Language Walkthrough Slides](http://localhost:4000/learn/language-concepts/Ballerina_Language_Presentation-2021-03-08.pdf)
```

Comment:
Why localhost here? And it seems `Language Introduction Slides` and `Language Walkthrough Slides` both referring to the same .pdf. Why is that?

Reply:
They were added just to complete the structure until we have 2 videos. Anyway, removed it and fixed the link.